### ISSUE TITLE
Marketplace Comments Error OpenCart Admin Dashboard

### ISSUE THREAD

**Issue by hiphopservers** [opencart/opencart#8955]
**What version of OpenCart are you reporting this for?**
Version 3.0.3.6

**Describe the bug**
When login admin browsing the marketplace.  I am not able to view comments from OpenCart Marketplace from my website's admin dashboard.  

**To Reproduce**
1. Go to 'Extensions'
2. Click on 'Click on Marketplace'
3. Scroll down to 'ANY extension with comments.'
4. Click on the extension and navigate to the comments tab.

**Expected behavior**
I should see the comments by users regarding the plugin and be allowed to post a comment.

**Screenshot**

![Screen Shot 2020-12-08 at 2 35 45 AM](https://user-images.githubusercontent.com/15718381/101453951-28114080-38fe-11eb-8c86-1469ae394af4.png)

**Server / Test environment (please complete the following information):**
 - Deployed to a web server.
```
Server Environment

Server Info: Apache/2
PHP Version: 7.4.13
PHP Post Max Size: 64 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.73.0
OpenSSL/1.0.2k-fips

SUHOSIN Installed: –
MySQL Version: 5.7.31
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔ - Version [e.g. 22]
```

**Comment by WebkulOpencart** [comment]
@hiphopservers Yes, I found the same also the marketplace API not returning the response.

@danielkerr Please check, marketplace API not returning the response.
[https://prnt.sc/w6lc06](https://prnt.sc/w6lc06)

**Comment by hiphopservers** [comment]
I spent way too much time configuring this application for it not work properly with multiple plugins or limited futures.  It seems most of the developers are focus on v2.x.x.x.x rather than version 3.0.x.x.

**Comment by prabhatkumaroc082** [comment]
fix #9036
